server/controllers/oneEvent.js

```diff
@@ -0,0 +1,33 @@
+const {
+  Event, Coffee, Lunch, Equipment,
+} = require('../database/models');
+
+exports.oneEvent = ((req, res, next) => {
```

Comment:
It would be better if you define the name of function better than ```oneEven```! call it getEvent

Reply:
done :+1: